renderer_test.go

```diff
@@ -33,3 +34,15 @@ func TestRendererWithOutput(t *testing.T) {
 		t.Error("Expected renderer to use true color")
 	}
 }
+
+func TestRace(t *testing.T) {
+	r := NewRenderer(io.Discard)
+
+	for i := 0; i < 100; i++ {
+		t.Run("SetColorProfile", func(t *testing.T) {
+			t.Parallel()
```

Comment:
I believe the parent test needs to call parallel too

Reply:
Not really, `t.Parallel` makes the test run parallel to its _adjacent_ tests. In this case, we want the 100 runs to run in parallel but not the parent with other tests.